deploy-service/common/src/main/java/com/pinterest/deployservice/db/DBHostDAOImpl.java

```diff
@@ -51,7 +51,7 @@ public class DBHostDAOImpl implements HostDAO {
     private static final String GET_GROUP_SIZE = "SELECT COUNT(host_id) FROM hosts WHERE group_name=?";
     private static final String GET_ALL_HOSTS_BY_GROUP = "SELECT * FROM hosts WHERE group_name=? AND state!='TERMINATING'";
     private static final String GET_HOST_BY_NAME = "SELECT * FROM hosts WHERE host_name=?";
-    private static final String GET_HOST_BY_HOSTID = "SELECT * FROM hosts WHERE host_id=?";
+    private static final String GET_HOST_BY_HOSTID = "SELECT * FROM hosts WHERE host_id=? ORDER BY create_date";
```

Comment:
Any reason to update this?

Reply:
Yes. The current order is indeterministic. The new usage [bellow](https://github.com/pinterest/teletraan/pull/1426/files#diff-067abe2f4de3c1b2b383cf5d982f41aa5d8c8232b22d68b5b617549816b70698R232) will benefit from the ordering. I also just code references and no existing code depend on the order of the query.